Modules/_threadmodule.c

```diff
@@ -1117,24 +1071,14 @@ rlock_acquire_restore(PyObject *op, PyObject *args)
     rlockobject *self = (rlockobject*)op;
     PyThread_ident_t owner;
     unsigned long count;
-    int r = 1;
 
     if (!PyArg_ParseTuple(args, "(k" Py_PARSE_THREAD_IDENT_T "):_acquire_restore",
```

Comment:
Would be good to use `size_t`  here (or `Py_ssize_t` if `size_t` is not practical).